.github/workflows/publish-docs-component.yml

```diff
@@ -96,5 +96,6 @@ jobs:
             components/console_simple_init;
             components/console_cmd_ping;
             components/console_cmd_ifconfig;
+            components/console_cmd_wifi;
```

Comment:
This will publish the new component.

Either remove this line or add a bump commit